package.json

```diff
@@ -81,14 +81,15 @@
     "is-port-reachable": "3.1.0",
     "jest": "27.5.1",
     "jscodeshift": "0.13.1",
-    "lerna": "4.0.0",
+    "lerna": "5.1.0-alpha.0",
```

Comment:
Looks like this is out of alpha now, and they're onto `5.1.1`.

Reply:
yeah, came here to say that 👍